chat-plugins/mafia.js

```diff
@@ -355,7 +355,7 @@ class MafiaTracker extends Rooms.RoomGame {
 		if (!player && this.dead[user.userid] && this.dead[user.userid].restless) player = this.dead[user.userid];
 		if (!(target in this.players) && target !== 'nolynch') return false;
 		if (player.lynching || (target === player.userid && !this.selfEnabled)) return false;
-		if (target === player.userid && (this.lynches[target] && this.getHammer() - 1 > this.lynches[target].count) && this.selfEnabled === 'hammer') return false;
+		if (target === player.userid && (this.lynches[target] && this.hammerCount - 1 > this.lynches[target].count) && this.selfEnabled === 'hammer') return false;
```

Comment:
Change this to `if (target === player.userid && (this.hammerCount - 1 > (this.lynches[target] ? this.lynches[target].count : 0)) && this.selfEnabled === 'hammer') return false;` (it fixes ppl lynching themselves when lynch obj doesnt exist of theirs)